### PR TITLE
upgrades from lvm 3.x latest to lvm 4.x latest

### PR DESCRIPTION
**Pull request by Ameenasuhani**
Description: upgrades from lvm 3.x latest to lvm 4.x latest

CLI used: python run.py --rhbuild 4.1-rhel-7 --global-conf conf/nautilus/upgrades/upgrade_3.x_to_4.x.yaml --osp-cred osp/osp-cred.yaml --inventory conf/inventory/rhel-7.8-server-x86_64.yaml --suite suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml --log-level info --store --ignore-latest-container

Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1589360951869/

### CODE REVIEW

**Comment by XDsubhash** [comment]
@Ameenasuhani can you add the cli to QE CI jenkins doc , helpers section (under upgrades)

File: suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            mon_max_pg_per_osd: 4096
+            mon_allow_pool_delete: True
+          client:
+            rgw crypt require ssl: false
+            rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+              testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+    desc: test cluster setup using ceph-ansible
+    abort-on-fail: True
+- test:
+    name: ceph ansible
```
Comment (XDsubhash) [inline suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml:37]: again #95 , have something like "upgrade with ceph-ansible"

**Comment by XDsubhash** [comment]
@Ameenasuhani as suggested in #101 can you rename the implicit ceph-disk to ceph-volume suite?

File: suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            osd_pool_default_pg_num: 64
+            osd_default_pool_size: 2
+            osd_pool_default_pgp_num: 64
+            mon_max_pg_per_osd: 1024
+            mon_allow_pool_delete: true
+            client:
+            rgw crypt require ssl: false
+            rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+              testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+    desc: Test Ceph-Ansible rolling update - 3.x -> 4.x
```
[the remark this answers is not in the document]
Comment (Ameenasuhani) [reply suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml:70]: @XDsubhash the rhel version is passed through cli. so the description, in this case, looks good to me.

[the remark this answers is not in the document]
Comment (XDsubhash) [reply suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml:70]: @Ameenasuhani how will you figure out from log which rhel version you upgraded? can you **point** me to the log where we can differentiate rhel versions before and after upgrade? 

lets say rhel7 to rhel8 upgrade scenarios are added to cephci , how will you know which is which ? I would ask you to include rhel version in suite file name too so it'll be helpful 

if you have any better suggestions pls let me know.

File: suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            mon_allow_pool_delete: True
+          client:
+            rgw crypt require ssl: false
+            rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+              testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+    desc: test cluster setup using ceph-ansible
+    abort-on-fail: True
+- test:
+    name: ceph ansible
+    module: test_ansible_upgrade.py
```
Comment (XDsubhash) [inline suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml:38]: polarion ID missing, check if there is one for normal upgrade or any new polarion ID included as part of rhcs4.x plan

Comment (rakeshgm) [reply suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml:37]: @Ameenasuhani could you add the polariton id here as we discussed. let's try to close this as fast as possible. its been pending for some time now.

File: suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ameenasuhani) [reply suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml:69]: okay I will do the changes

**Comment by Ameenasuhani** [comment]
closing this pr due to rebase issue 
resolved all the changes mentioned here in new pr: https://github.com/red-hat-storage/cephci/pull/127

**Pull request closed without merging**